packages/graphql-language-service-server/src/GraphQLWatchman.js

```diff
@@ -88,7 +91,7 @@ export class GraphQLWatchman {
     const {watch, relative_path} = await this.watchProject(entryPath);
 
     // Subscribe to the relative path
```

Comment:
Is this comment out of date now that we are subscribing to `relative_path || watch`?